addon/components/polaris-resource-list/filter-control/filter-value-selector.js

```diff
@@ -123,8 +118,17 @@ export default Component.extend({
     }
 
     this.handleOperatorOptionChange(operatorText[0].key);
-  },
-});
+  }
+
+  init() {
+    super.init(...arguments);
+
+    // Using `afterRender` here instead of `didInsertElement` hook.
+    // This component is tagless and has no container div so we can't
+    // use `{{did-insert}}` modifiers in the template.
+    scheduleOnce('afterRender', this, this.insertFilterValueSelector);
```

Comment:
I wanted to specifically point out this change since it feels kind of like a hax, but we get double-render errors since this method (the contents of `insertFilterValueSelector`) was originally moved from `didInsertElement` to `init`. We need to wait until the element is rendered to run this and this feels like the best way to achieve a `didInsertElement` without actually using the hook 😬 

This is tagless and the main blocks of the template are `<PolarisStack>`s, so we can't have `{{did-insert}}` modifiers on them because they're components (it works, but tests will error since a modifier on a component is not really acceptable and a bad practice).

Reply:
Interesting, thanks for the explanation @tomnez! Maybe worth adding as a comment, unless @vladucu has a suggestion for getting around this?

Reply:
@tomnez I don't think it's that bad what you have here
one thought we could check....is to just have a simple `<span {{did-insert ...}}></span>` element added (the span should not cause any spacing issues I think)

Reply:
Yeah that works too 👍I wasn't sure if we wanted to go adding extra nodes that weren't in the React version but I'd rather use the `span` if we're all cool with it. Thanks ya'll.